libsolc/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(
```

Comment:
You forgot to include this script in main CMakeLists.txt script. CMake does not know what `libsolc` is and it assumed its some system library.